src/main/java/io/supertokens/webserver/api/bulkimport/BulkImportBackgroundJobManagerAPI.java

```diff
@@ -0,0 +1,110 @@
+/*
+ *    Copyright (c) 2024, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ *    This software is licensed under the Apache License, Version 2.0 (the
+ *    "License") as published by the Apache Software Foundation.
+ *
+ *    You may not use this file except in compliance with the License. You may
+ *    obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *    License for the specific language governing permissions and limitations
+ *    under the License.
+ */
+
+package io.supertokens.webserver.api.bulkimport;
+
+import com.google.gson.JsonObject;
+import io.supertokens.Main;
+import io.supertokens.bulkimport.BulkImport;
+import io.supertokens.bulkimport.BulkImportBackgroundJobManager;
+import io.supertokens.pluginInterface.multitenancy.exceptions.TenantOrAppNotFoundException;
+import io.supertokens.storageLayer.StorageLayer;
+import io.supertokens.webserver.InputParser;
+import io.supertokens.webserver.WebserverAPI;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.io.Serial;
+
+public class BulkImportBackgroundJobManagerAPI extends WebserverAPI {
+
+    @Serial
+    private static final long serialVersionUID = 2380841048110043408L;
+
+    public BulkImportBackgroundJobManagerAPI(Main main) {
+        super(main, "");
+    }
+
+    @Override
+    public String getPath() {
+        return "/bulk-import/backgroundjob";
+    }
+
+    //TODO what's stopping two independent calls interfering with each-other?
```

Comment:
TODO

Reply:
yeah, but I don't have an answer for this. I can remove the comment but I don't have a solution for the written problem, apart from the clients who are orchestrating the imports should be aware. So, should I remove the comment? Do you have a better idea for solving this?

Reply:
maybe `startBackgroundJob` and`stopBackgroundJob` should use a shared lock?